src/autodev.ts

```diff
@@ -183,6 +181,13 @@ const merge = async (
     `The following branches have been merged:\n${successList}\n\n` +
     `The following branches failed to merge:\n${failList}`
 
+  if (success.length === 0) {
+    return message
+  }
```

Comment:
This is the important change to exit the action if there was no successful merge. 🙌